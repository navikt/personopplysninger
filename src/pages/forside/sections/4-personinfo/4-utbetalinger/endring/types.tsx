import { OptionType } from '../../../../../../types/option';

export interface OutboundNorskKontonummer {
    value: string;
}

export interface OutboundUtenlandsbankonto {
    value: string | null;
    utenlandskKontoInformasjon: {
        landkode: string;
        landkodeTobokstavs?: string;
        valuta: string;
        swift?: string;
        bank: {
            adresseLinje1?: string;
            adresseLinje2?: string;
            adresseLinje3?: string;
            kode?: string;
            navn: string;
        };
    };
}

export interface FormFields {
    kontonummer: string;
    land?: OptionType;
    valuta?: OptionType;
    banknavn: string;
    kontonummerIban: string;
    bickode: string;
    retningsnummer: string;
    bankkode: string;
    adresse1: string;
    adresse2: string;
    adresse3: string;
}
