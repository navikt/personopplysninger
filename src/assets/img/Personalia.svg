<?xml version="1.0" encoding="UTF-8"?>
<svg width="64px" height="64px" viewBox="0 0 64 64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 55 (78076) - https://sketchapp.com -->
    <title>Group 8@1x</title>
    <desc>Created with Sketch.</desc>
    <defs>
        <rect id="path-1" x="0" y="0" width="64" height="64"></rect>
        <circle id="path-3" cx="32" cy="32" r="32"></circle>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="GDPR-Personopplysning" transform="translate(-595.000000, -568.000000)">
            <g id="Group-8" transform="translate(595.000000, 568.000000)">
                <g id="spotlight/-gjenstand-">
                    <mask id="mask-2" fill="white">
                        <use xlink:href="#path-1"></use>
                    </mask>
                    <g id="Background"></g>
                    <g id="spotlight/-element" mask="url(#mask-2)" fill-rule="evenodd" stroke-width="1">
                        <g id="ðŸŒˆ-60">
                            <mask id="mask-4" fill="white">
                                <use xlink:href="#path-3"></use>
                            </mask>
                            <g id="Mask"></g>
                            <g id="NAV-Brand/farge/-BlÃ¥/-60" mask="url(#mask-4)" fill="#99C2E8">
                                <polygon id="blÃ¥-60" points="0 64 64 64 64 0 0 0"></polygon>
                            </g>
                        </g>
                    </g>
                </g>
                <g id="Group-30" transform="translate(12.800000, 19.200000)">
                    <path d="M37.6,0 L0.8,0 C0.3584,0 0,0.3584 0,0.8 L0,26.4 C0,26.8416 0.3584,27.2 0.8,27.2 L7.2,27.2 C7.6416,27.2 8,26.8416 8,26.4 L8,25.2 C8,24.0976 8.8976,23.2 10,23.2 C11.1024,23.2 12,24.0976 12,25.2 L12,26.4 C12,26.8416 12.3584,27.2 12.8,27.2 L25.6,27.2 C26.0416,27.2 26.4,26.8416 26.4,26.4 L26.4,25.2 C26.4,24.0976 27.2976,23.2 28.4,23.2 C29.5024,23.2 30.4,24.0976 30.4,25.2 L30.4,26.4 C30.4,26.8416 30.7584,27.2 31.2,27.2 L37.6,27.2 C38.0416,27.2 38.4,26.8416 38.4,26.4 L38.4,0.8 C38.4,0.3584 38.0416,0 37.6,0 Z" id="line/id-card-1" fill="#FFFFFF" fill-rule="nonzero"></path>
                    <ellipse id="Oval" fill="#337C9B" cx="9.63529412" cy="8" rx="3.23529412" ry="3.2"></ellipse>
                    <path d="M16.1411765,19.2 C16.1411765,15.6653776 13.2441954,12.8 9.67058824,12.8 C6.09698103,12.8 3.2,15.6653776 3.2,19.2 C3.2,19.2 15.7098039,19.2 16.1411765,19.2 Z" id="Oval" fill="#337C9B"></path>
                    <path d="M19.8545455,17.6 L32.9454545,17.6 C33.30695,17.6 33.6,17.9581722 33.6,18.4 C33.6,18.8418278 33.30695,19.2 32.9454545,19.2 L19.8545455,19.2 C19.49305,19.2 19.2,18.8418278 19.2,18.4 C19.2,17.9581722 19.49305,17.6 19.8545455,17.6 Z" id="Rectangle" fill="#3E3832"></path>
                    <path d="M19.8545455,14.4 L32.9454545,14.4 C33.30695,14.4 33.6,14.7581722 33.6,15.2 C33.6,15.6418278 33.30695,16 32.9454545,16 L19.8545455,16 C19.49305,16 19.2,15.6418278 19.2,15.2 C19.2,14.7581722 19.49305,14.4 19.8545455,14.4 Z" id="Rectangle" fill="#3E3832"></path>
                    <path d="M19.8545455,11.2 L32.9454545,11.2 C33.30695,11.2 33.6,11.5581722 33.6,12 C33.6,12.4418278 33.30695,12.8 32.9454545,12.8 L19.8545455,12.8 C19.49305,12.8 19.2,12.4418278 19.2,12 C19.2,11.5581722 19.49305,11.2 19.8545455,11.2 Z" id="Rectangle" fill="#3E3832"></path>
                    <path d="M19.84,6.4 L24.96,6.4 C25.3134622,6.4 25.6,6.7581722 25.6,7.2 C25.6,7.6418278 25.3134622,8 24.96,8 L19.84,8 C19.4865378,8 19.2,7.6418278 19.2,7.2 C19.2,6.7581722 19.4865378,6.4 19.84,6.4 Z" id="Rectangle" fill="#3E3832"></path>
                </g>
            </g>
        </g>
    </g>
</svg>