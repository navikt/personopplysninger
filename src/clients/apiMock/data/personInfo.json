{
  "personalia": {
    "fornavn": "DONALD FAUNTLEROY",
    "etternavn": "DUCK",
    "personident": {
      "verdi": "12345678911",
      "type": "FNR"
    },
    "utenlandskbank": {
      "adresse1": "Gävlegatan 22",
      "adresse2": "113 30 Stockholm",
      "adresse3": "",
      "bankkode": "90081210188",
      "banknavn": "Nordax Bank AB",
      "iban": "",
      "kontonummer": "US90081210188",
      "land": "Sverige",
      "swiftkode": "HELGSE21",
      "valuta": "Tugriks Mongolia"
    },
    "tlfnr": {
      "jobb": "12121212",
      "mobil": "24242424",
      "privat": "36363636",
      "datakilder": [{}, {}]
    },
    "spraak": "NB",
    "epostadr": "TODO",
    "personstatus": "BOSA",
    "statsborgerskap": "NOR",
    "foedested": "NOR",
    "sivilstand": "UGIF",
    "kjoenn": "M",
    "datakilder": [{}, {}, {}, {}]
  },
  "enhetKontaktInformasjon": {
    "enhet": {
      "postadresse": {
        "type": "stedsadresse",
        "postnummer": "0614",
        "poststed": "Oslo",
        "gatenavn": "Slottsveien",
        "husnummer": "22",
        "husbokstav": "A"
      },
      "publikumsmottak": [
        {
          "gateadresse": "Peder Falcks veg",
          "poststed": "Trondheim",
          "husnummer": "31",
          "husbokstav": null,
          "postnummer": "7044",
          "stedsbeskrivelse": null,
          "aapningMandag": {
            "dag": "Mandag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": "Timeavtaler også utenom åpningstiden"
          },
          "aapningTirsdag": {
            "dag": "Tirsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningOnsdag": {
            "dag": "Onsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningTorsdag": {
            "dag": "Torsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningFredag": {
            "dag": "Fredag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningAndre": [
            {
              "dag": "15.05.2019",
              "fra": "09:00",
              "til": "12:00",
              "stengt": "false",
              "kommentar": "Her kan det også være en merknad "
            },
            {
              "dag": "17.05.2019",
              "fra": null,
              "til": null,
              "stengt": "true"
            },
            {
              "dag": "29.05.2019",
              "fra": "09:00",
              "til": "13:30",
              "stengt": "false",
              "kommentar": "Onsdag før Kristi himmelfartsdag"
            }
          ]
        },
        {
          "gateadresse": "Kirkegata",
          "poststed": "Trondheim",
          "husnummer": "25",
          "husbokstav": null,
          "postnummer": "7014",
          "stedsbeskrivelse": null,
          "aapningMandag": {
            "dag": "Mandag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": "Timeavtaler også utenom åpningstiden"
          },
          "aapningTirsdag": {
            "dag": "Tirsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningOnsdag": {
            "dag": "Onsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningTorsdag": {
            "dag": "Torsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningFredag": {
            "dag": "Fredag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningAndre": [
            {
              "dag": "15.05.2019",
              "fra": "09:00",
              "til": "12:00",
              "stengt": "false",
              "kommentar": "Her kan det også være en merknad "
            },
            {
              "dag": "17.05.2019",
              "fra": null,
              "til": null,
              "stengt": "true"
            },
            {
              "dag": "29.05.2019",
              "fra": "09:00",
              "til": "13:30",
              "stengt": "false",
              "kommentar": "Onsdag før Kristi himmelfartsdag"
            }
          ]
        }
      ],
      "tlfperson": "55553333",
      "spesielleopplysninger": "Timeavtale 8-15, man - fre."
    }
  },
  "adresser": {
    "boadresse": {
      "adressetillegg": "C/O Rune Aremark",
      "adresse": "Bjørvika 4D",
      "datoFraOgMed": "2017-02-21",
      "kommune": "1239",
      "land": "Norge",
      "matrikkeladresse": {},
      "postnummer": "1239",
      "poststed": "Jepp",
      "veiadresse": {
        "bokstav": "A",
        "bolignummer": "H0102",
        "gatekode": "01239",
        "husnummer": "11"
      }
    },
    "geografiskTilknytning": {
      "bydel": "500102",
      "datoFraOgMed": "2012-10-01",
      "kommune": null,
      "land": null,
      "enhet": "NAV Falkenborg"
    },
    "postadresse": {
      "adresse1": "12345 HFJEF IQUFJ HFURHGHQBEUQ",
      "adresse2": "BATOOM LUNCHAGRAMA,GONGRUPEA,DD",
      "adresse3": "GGLAM,10000 HCAMGHU,THAILAND",
      "datoFraOgMed": "2018-03-13",
      "land": "THA",
      "postnummer": "",
      "poststed": "Nittedal"
    },
    "prioritertAdresse": "TIAD",
    "tilleggsadresse": {
      "adresse1": "JEPP 10jfioew JIO LPFL ",
      "adresse2": "LOOOL FKEOWIF JEWIOF WECOOOO FJEI",
      "adresse3": "",
      "bolignummer": "",
      "bydel": "",
      "datoFraOgMed": "2019-01-12",
      "datoTilOgMed": "2023-01-11",
      "gateKode": "0270",
      "husbokstav": "",
      "husnummer": "0020",
      "kommunenummer": "0131",
      "postboksanlegg": "",
      "postboksnummer": "",
      "postnummer": "2021",
      "poststed": "JAJAJA"
    },
    "datakilder": [{}, {}]
  }
}
