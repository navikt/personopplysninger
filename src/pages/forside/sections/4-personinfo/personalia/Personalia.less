.underseksjon {
  &__beskrivelse {
    padding: 1rem 0 0 0;
  }
}
