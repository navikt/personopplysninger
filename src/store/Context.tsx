import { createContext, useContext, useReducer, Dispatch } from 'react';
import { Action, initialState, reducer, Store } from './Store';
interface Props {
    children: JSX.Element | JSX.Element[];
}

export const StoreContext = createContext({} as [Store, Dispatch<Action>]);
export const StoreProvider = (props: Props) => {
    const { children } = props;

    return <StoreContext.Provider value={useReducer(reducer, initialState)}>{children}</StoreContext.Provider>;
};
export const useStore = () => useContext(StoreContext);
