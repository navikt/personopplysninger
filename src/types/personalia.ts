export interface Tlfnr {
    telefonHoved?: string;
    telefonAlternativ?: string;
    landskodeHoved?: string;
    landskodeAlternativ?: string;
}

export interface UtenlandskBankkonto {
    adresse1: string;
    adresse2: string;
    adresse3: string;
    bankkode: string;
    banknavn: string;
    iban: string;
    kontonummer: string;
    land: string;
    swiftkode: string;
    valuta: string;
}

export interface Personalia {
    fornavn?: string;
    etternavn?: string;
    personident?: {
        verdi: string;
        type: string;
    };
    kontoregisterStatus: string;
    kontonr?: string;
    utenlandskbank: UtenlandskBankkonto;
    tlfnr?: Tlfnr;
    statsborgerskap?: string[];
    foedested?: string;
    sivilstand?: string;
    kjoenn?: string;
}
