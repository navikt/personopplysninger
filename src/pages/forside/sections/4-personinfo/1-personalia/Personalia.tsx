import React, { Component } from "react";
import Box from "../../../../../components/box/Box";
import personaliaIkon from "../../../../../assets/img/Personalia.svg";
import { Personalia as PersonaliaType } from "../../../../../types/personalia";
import ListElement from "../../../../../components/listelement/ListElement";
import Kilde from "../../../../../components/kilde/Kilde";
import eksternLenkeIkon from "../../../../../assets/img/Link.svg";

interface Props {
  personalia: PersonaliaType;
}

class Personalia extends Component<Props> {
  render() {
    const {
      personident,
      fornavn,
      etternavn,
      spraak,
      statsborgerskap,
      foedested,
      sivilstand,
      kjoenn
    } = this.props.personalia;

    const personidentHeader =
      personident && personident.type === "DNR"
        ? "personalia.dnr"
        : "personalia.fnr";

    const fornavnHeader =
      fornavn && fornavn.indexOf(" ") === -1
        ? "personalia.first_name"
        : "personalia.first_and_middle_name";

    const formattertPersonident =
      personident && personident.verdi.length === 11
        ? {
            ...personident,
            verdi: personident.verdi.replace(/^(.{6})(.*)$/, "$1 $2")
          }
        : personident;

    return (
      <Box
        id="personalia"
        tittel="personalia.tittel"
        beskrivelse="personalia.beskrivelse"
        icon={personaliaIkon}
      >
        <hr className="box__linje-bred" />
        <ul className="list-column-2">
          <ListElement titleId={fornavnHeader} content={fornavn} />
          <ListElement
            className="capitalize"
            titleId="personalia.surname"
            content={etternavn}
          />
          {formattertPersonident && (
            <ListElement
              titleId={personidentHeader}
              content={formattertPersonident.verdi}
            />
          )}
          <ListElement titleId="personalia.language" content={spraak} />
          <ListElement
            titleId="personalia.citizenship"
            content={statsborgerskap}
          />
          <ListElement titleId="personalia.birthplace" content={foedested} />
          <ListElement titleId="personalia.civil_status" content={sivilstand} />
          <ListElement titleId="personalia.gender" content={kjoenn} />
        </ul>
        <Kilde
          kilde="personalia.source.folkeregisteret"
          lenke="https://www.skatteetaten.no/person/folkeregister/"
          lenkeTekst="personalia.link.folkeregisteret"
          eksternLenke={true}
          ikon={eksternLenkeIkon}
        />
      </Box>
    );
  }
}

export default Personalia;
