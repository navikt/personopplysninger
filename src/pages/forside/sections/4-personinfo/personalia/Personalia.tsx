import React, { Component } from "react";
import Box from "../../../../../components/box/Box";
import personalia from "../../../../../assets/img/personalia.svg";
import Kontonummer from "./subsections/Kontonummer";
import Folkeregisteret from "./subsections/Folkeregisteret";
import Telefonnummer from "./subsections/Telefonnummer";
import DKIF from "./subsections/DKIF";
import { Personalia as PersonaliaType } from "../../../../../types/personalia";

interface Props {
  personalia: PersonaliaType;
}

class Personalia extends Component<Props> {
  render() {
    const { kontonr, utenlandskbank } = this.props.personalia;

    return (
      <Box
        id="personalia"
        tittel="personalia.tittel"
        beskrivelse="personalia.beskrivelse"
        icon={personalia}
      >
        <hr className="box__linje-bred" />
        <Folkeregisteret personalia={this.props.personalia} />
        <Kontonummer kontonummer={kontonr} utenlandskBankkonto={utenlandskbank} />
        <Telefonnummer tlfnr={this.props.personalia.tlfnr} />
        <DKIF />
      </Box>
    );
  }
}

export default Personalia;
