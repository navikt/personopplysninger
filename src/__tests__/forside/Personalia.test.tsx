import '@testing-library/jest-dom';
import { IntlProvider } from 'react-intl';
import nbMessages from 'text/nb';
import personInfo from '../../clients/apiMock/app/fetch/person-info.json';
import Personalia from '../../pages/forside/sections/4-personinfo/1-personalia/Personalia';
import { Personalia as PersonaliaType } from 'types/personalia';
import { StoreProvider } from '../../store/Context';
import { render } from '@testing-library/react';

jest.mock('react-modal');

describe('Personalia', () => {
    it('renders correctly', () => {
        const { asFragment } = render(
            <StoreProvider>
                <IntlProvider locale={'nb'} messages={nbMessages}>
                    <Personalia personalia={personInfo as unknown as PersonaliaType} />
                </IntlProvider>
            </StoreProvider>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
