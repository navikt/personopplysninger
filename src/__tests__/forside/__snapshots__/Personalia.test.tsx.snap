// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Personalia renders correctly 1`] = `
<DocumentFragment>
  <div
    class="box__wrapper"
    id="personalia"
  >
    <div
      class="navds-guide-panel navds-guide-panel--poster"
    >
      <div
        class="navds-guide"
      >
        <img
          alt=""
          class="box__ikon"
          src="Personalia.svg"
        />
      </div>
      <div
        class="navds-guide-panel__content"
      >
        <div
          class="box__container"
        >
          <div
            class="box__header"
          >
            <div
              class="box__title-container"
            >
              <div
                class="box__line"
              />
              <h2
                class="box__title navds-heading navds-heading--medium"
              >
                Personalia
              </h2>
              <button
                aria-label="Les mer om denne seksjonen."
                aria-pressed="false"
                class="infotekst__title-i-button"
              >
                <svg
                  aria-hidden="true"
                  class="infotekst__title-i-icon"
                  fill="none"
                  focusable="false"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M12 5.5a1 1 0 1 0 0 2 1 1 0 0 0 0-2Zm-2.75 4a.75.75 0 0 1 .75-.75h2a.75.75 0 0 1 .75.75v7.25H14a.75.75 0 0 1 0 1.5h-4a.75.75 0 0 1 0-1.5h1.25v-6.5H10a.75.75 0 0 1-.75-.75Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </button>
              <dialog
                aria-labelledby=":r0:"
                class="navds-modal infotekst__modal navds-modal--polyfilled navds-modal--autowidth"
                role="dialog"
              >
                <div
                  class="navds-modal__header"
                >
                  <button
                    class="navds-modal__button navds-button navds-button--tertiary-neutral navds-button--small navds-button--icon-only"
                    type="button"
                  >
                    <span
                      class="navds-button__icon"
                    >
                      <svg
                        aria-labelledby="title-r4"
                        fill="none"
                        focusable="false"
                        height="1em"
                        role="img"
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <title
                          id="title-r4"
                        >
                          Lukk
                        </title>
                        <path
                          d="M6.53 5.47a.75.75 0 0 0-1.06 1.06L10.94 12l-5.47 5.47a.75.75 0 1 0 1.06 1.06L12 13.06l5.47 5.47a.75.75 0 1 0 1.06-1.06L13.06 12l5.47-5.47a.75.75 0 0 0-1.06-1.06L12 10.94 6.53 5.47Z"
                          fill="currentColor"
                        />
                      </svg>
                    </span>
                  </button>
                  <h2
                    class="navds-heading navds-heading--small"
                    id=":r0:"
                  >
                    Personalia
                  </h2>
                </div>
                <div
                  class="navds-modal__body"
                >
                  <div
                    class="infotekst__ingress"
                  >
                    <p
                      class="navds-body-long navds-body-long--medium"
                    >
                      Vi må vite hvem du er og hvordan vi kan kontakte deg. Hvis du skal ha tjenester eller søker om økonomisk støtte, trenger vi opplysninger for å kunne vurdere hva du har krav på. For å kunne utbetale penger trenger vi kontonummeret ditt og en adresse.
                    </p>
                  </div>
                </div>
              </dialog>
              <div
                class="box__line"
              />
            </div>
            <a
              class="navds-link anchor-link navds-link--action"
              href="#personalia"
            >
              <svg
                aria-hidden="true"
                class="anchor-link__icon"
                fill="none"
                focusable="false"
                height="1em"
                role="img"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M14.909 5.152a2.25 2.25 0 0 1 3.182 0l.757.757a2.25 2.25 0 0 1 0 3.182l-4.964 4.965a1.25 1.25 0 0 1-1.768 0L11.03 12.97a.75.75 0 1 0-1.06 1.06l1.085 1.086a2.75 2.75 0 0 0 3.89 0l4.964-4.964a3.75 3.75 0 0 0 0-5.304l-.757-.757-.53.53.53-.53a3.75 3.75 0 0 0-5.304 0l-.878.879a.75.75 0 0 0 1.06 1.06l.879-.878Zm-1.964 3.732a2.75 2.75 0 0 0-3.89 0l-4.964 4.964a3.75 3.75 0 0 0 0 5.304l.513-.514-.513.514.757.757a3.75 3.75 0 0 0 5.304 0l.878-.879a.75.75 0 0 0-1.06-1.06l-.879.878a2.25 2.25 0 0 1-3.182 0l-.757-.757a2.25 2.25 0 0 1 0-3.182l4.964-4.965a1.25 1.25 0 0 1 1.768 0l1.086 1.086a.75.75 0 0 0 1.06-1.06l-1.085-1.086Z"
                  fill="currentColor"
                  fill-rule="evenodd"
                />
              </svg>
              Lenke hit
            </a>
          </div>
          <dl
            class="list"
          />
          <div
            class="kilde__tekst"
          >
            <p
              class="navds-detail navds-typo--spacing"
            >
              Kilde: Folkeregisteret
            </p>
          </div>
          <p
            class="navds-body-long navds-body-long--medium"
          >
            <a
              class="navds-link navds-link--action"
              href="https://www.skatteetaten.no/person/folkeregister/"
            >
              <span
                class="kilde__icon"
              >
                <svg
                  aria-hidden="true"
                  class="kilde__icon-aksel"
                  fill="none"
                  focusable="false"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M20.532 3.471A.748.748 0 0 1 20.75 4v7.5a.75.75 0 0 1-1.5 0V5.81l-8.72 8.72a.75.75 0 1 1-1.06-1.06l8.72-8.72H12.5a.75.75 0 0 1 0-1.5H20c.206 0 .393.083.529.218l.001.002.002.001ZM4.75 9A.25.25 0 0 1 5 8.75h7a.75.75 0 0 0 0-1.5H5A1.75 1.75 0 0 0 3.25 9v10c0 .966.784 1.75 1.75 1.75h10A1.75 1.75 0 0 0 16.75 19v-7a.75.75 0 0 0-1.5 0v7a.25.25 0 0 1-.25.25H5a.25.25 0 0 1-.25-.25V9Z"
                    fill="currentColor"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
              Endre i Folkeregisteret
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
